scalableminds/webknossos
# Honor mappingStatus when requesting to compute meshes ad-hoc

The `mappingName` and `mappingType` remain in the store if a mapping is disabled. Therefore, the `mappingStatus` needs to be taken into account to check whether a mapping is actually enabled.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Open a dataset with a segmentation layer and a mapping
- Compute an ad-hoc mesh for a segment (using the context-menu or the "Segments" list - segments appear in the segment list after clicking on them)
- Enable a mapping and compute an ad-hoc mesh for the mapped segment
- Disable the mapping again and delete both meshes in the "Segments" list
- Compute an ad-hoc mesh for the first segment -> The result should be the same as in the beginning

### Issues:
- fixes #5966 

------
(Please delete unneeded items, merge only when none are left open)
